### PR TITLE
Update supported-providers.md

### PR DESCRIPTION
**Pull request by bedouin345**
## Breaking change

```txt
<placeholder>
```

## Proposed change

```txt
<placeholder>
```

## Type of change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to a this container)
- [ ] Breaking change (fix/feature causing existing functionality to break)

## Additional information

- This PR fixes or closes issue: ```fixes #```
- This PR is related to issue: ```relates to #```
- Link to documentation updated (if done separately): ```https://...```

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.

If user exposed functionality or configuration variables are added/changed:

- [x] Documentation added/updated

### CODE REVIEW

File: docs/supported-providers.md [hunk 1]
```diff
@@ -80,14 +84,42 @@ If you are using AirVPN or other provider with credentials in the config file, y
 to set `OPENVPN_USERNAME` and `OPENVPN_PASSWORD` as this is required by the startup script.
 They will not be read by the .ovpn file, so you can set them to whatever.
 
-Note that you still need to modify your .ovpn file as described in the previous section.
-If you have an separate ca.crt, client.key or client.crt file in your volume mount should be a folder containing both the ca.crt and the .ovpn config.
+If `-e OPENVPN_CONFIG=` variable has been omitted from the `docker run` command the .ovpn config file must be named default.ovpn.
+If you have an separate ca.crt, client.key or client.crt file, then refer to the section below instead.
```
Comment (pkishino) [inline docs/supported-providers.md:88]: a separate

File: docs/supported-providers.md [hunk 2]
```diff
@@ -80,14 +84,42 @@ If you are using AirVPN or other provider with credentials in the config file, y
...
+Compose sample:
+```
+             - /volume1/docker/transmission/ipvanish/:/etc/openvpn/custom/
+```
+**Declare the Custom provider, the target server and login/password**
+Compose sample:
+```
+            - OPENVPN_PROVIDER=custom
+            - OPENVPN_CONFIG=ipvanish-UK-Maidenhead-lhr-c02
+            - OPENVPN_USERNAME=uourlogin
```
Comment (pkishino) [inline docs/supported-providers.md:108]: yourlogin..
although I dislike conjoints like this..
I think either use "user/pass" as elsewhere or <username>/<password>

**Review by pkishino — CHANGES_REQUESTED** [review]

File: docs/supported-providers.md (hunk 2 above)
Comment (bedouin345) [reply docs/supported-providers.md:108]: I'm completely ok with that change, I  don't see how to update the wiki though (very noob to Github as you may notice)

**Review by pkishino — APPROVED** [review]

**Comment by pkishino** [comment]
my bad..I'll fi manually..

**Pull request closed without merging**